### DISCUSSION TITLE
What is the recommended approach to migrate databases?

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#9377] [General]
Hi

what is the recommended approach when it comes to migrating the type of database used for the instance? Eg from SQLite to MSSQL. At the moment, I am exporting all content and settings, and doing and then creating a new instance from setup and then importing the stuff. 

Is there a better approach?

**Comment by hishamco** [comment]
The way I'm thinking is to use recipes, which is the one you pointed to if i'm not wrong
